docs/book/src/tasks/bootstrap/microk8s-bootstrap.md

```diff
@@ -0,0 +1,94 @@
+# Cluster API bootstrap provider MicroK8s
+## What is the Cluster API bootstrap provider MicroK8s?
+
+Cluster API bootstrap provider MicroK8s (CABPM) is a component responsible for generating a cloud-init script to turn a Machine into a Kubernetes Node. This implementation uses [MicroK8s](https://github.com/canonical/microk8s) for Kubernetes bootstrap.
+
+### Resources
+
+* [Design doc](https://github.com/canonical/cluster-api-control-plane-provider-microk8s/blob/main/README.md)
```

Comment:
```suggestion
* [CABPM Repository](https://github.com/canonical/cluster-api-bootstrap-provider-microk8s/blob/main/README.md)
```
I would change Design doc to e.g. "CABPM repository" (just because the readme isn't really a design doc)

(I assume it should point to the bootstrap provider)